gatsby-config.js

```diff
@@ -1,6 +1,7 @@
-require('dotenv').config({
-  path: `.env.${process.env.NODE_ENV}`,
-});
+// require('dotenv').config({
+//   path: `.env.${process.env.NODE_ENV}`,
+// });
+require("dotenv").config()
```

Comment:
I think this should be ok. Do we use `.env` files in any other environments? I'm pretty sure we don't. :+1: 

Could you delete the commented out code?